src/containers/TriggerTemplates/TriggerTemplates.js

```diff
@@ -0,0 +1,187 @@
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import isEqual from 'lodash.isequal';
+import { ALL_NAMESPACES, getErrorMessage } from '@tektoncd/dashboard-utils';
+import { InlineNotification } from 'carbon-components-react';
+import {
+  FormattedDate,
+  LabelFilter,
+  Table
+} from '@tektoncd/dashboard-components';
+import { fetchTriggerTemplates } from '../../actions/triggerTemplates';
+import {
+  getSelectedNamespace,
+  getTriggerTemplates,
+  getTriggerTemplatesErrorMessage,
+  isFetchingTriggerTemplates,
+  isWebSocketConnected
+} from '../../reducers';
+
+import './TriggerTemplates.scss';
+
+export /* istanbul ignore next */ class TriggerTemplates extends Component {
+  componentDidMount() {
+    this.fetchTriggerTemplates();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { filters, namespace, webSocketConnected } = this.props;
+    const {
+      filters: prevFilters,
+      namespace: prevNamespace,
+      webSocketConnected: prevWebSocketConnected
+    } = prevProps;
+
+    if (
+      !isEqual(filters, prevFilters) ||
+      namespace !== prevNamespace ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchTriggerTemplates();
+    }
+  }
+
+  handleAddFilter = labelFilters => {
+    const queryParams = `?${new URLSearchParams({
+      labelSelector: labelFilters
+    }).toString()}`;
+
+    const currentURL = this.props.match.url;
+    const browserURL = currentURL.concat(queryParams);
+    this.props.history.push(browserURL);
+  };
+
+  handleDeleteFilter = filter => {
+    const currentQueryParams = new URLSearchParams(this.props.location.search);
+    const labelFilters = currentQueryParams.getAll('labelSelector');
+    const labelFiltersArray = labelFilters.toString().split(',');
+    const index = labelFiltersArray.indexOf(filter);
+    labelFiltersArray.splice(index, 1);
+
+    const currentURL = this.props.match.url;
+    if (labelFiltersArray.length === 0) {
+      this.props.history.push(currentURL);
+    } else {
+      const newQueryParams = `?${new URLSearchParams({
+        labelSelector: labelFiltersArray
+      }).toString()}`;
+      const browserURL = currentURL.concat(newQueryParams);
+      this.props.history.push(browserURL);
+    }
+  };
+
+  fetchTriggerTemplates() {
+    const { filters, namespace } = this.props;
+    this.props.fetchTriggerTemplates({
+      filters,
+      namespace
+    });
+  }
+
+  render() {
+    const {
+      error,
+      filters,
+      loading,
+      selectedNamespace,
+      triggerTemplates
+    } = this.props;
+
+    const initialHeaders = [
+      { key: 'name', header: 'Name' },
```

Comment:
user-facing strings need to be externalised using react-intl, `intl.formatMessage(...)`

Reply:
I see the headers are updated but there are still other strings that need to be updated. For example:
- https://github.com/tektoncd/dashboard/pull/734/files#diff-a2df8045d49129a6538a03b327dd25e3R143
- https://github.com/tektoncd/dashboard/pull/734/files#diff-a2df8045d49129a6538a03b327dd25e3R145